<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-F9S85B0" failures="2" tests="6" name="Test" time="80.278" errors="0" timestamp="2025-03-09T06:40:18 IST">
  <testcase classname="testCases.TC001_AccountRegistrationTest" name="verify_accout_registraion" time="3.609"/>
  <testcase classname="testCases.TC002_LoginTest" name="verify_login" time="2.312"/>
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="6.23"/>
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="11.719">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC003_LoginDDT.verify_loginDDT(TC003_LoginDDT.java:60)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="13.168">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at testCases.TC003_LoginDDT.verify_loginDDT(TC003_LoginDDT.java:60)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- verify_loginDDT -->
  <testcase classname="testCases.TC003_LoginDDT" name="verify_loginDDT" time="6.81"/>
</testsuite> <!-- Test -->
